packages/react-components/react-overflow/stories/Overflow/PriorityWithDividers.stories.tsx

```diff
@@ -18,22 +35,111 @@ export const PriorityWithDividers = () => {
   return (
     <Overflow overflowDirection="start" padding={30}>
       <div className={styles.container}>
-        <TestOverflowItem id={'6'} priority={6} groupId={'1'} />
-        <TestOverflowGroupDivider groupId={'1'} />
-        <TestOverflowItem id={'7'} priority={7} groupId={'2'} />
-        <TestOverflowGroupDivider groupId={'2'} />
-        <TestOverflowItem id={'4'} priority={4} groupId={'3'} />
-        <TestOverflowItem id={'5'} priority={5} groupId={'3'} />
-        <TestOverflowGroupDivider groupId={'3'} />
-        <TestOverflowItem id={'1'} priority={1} groupId={'4'} />
-        <TestOverflowItem id={'2'} priority={2} groupId={'4'} />
-        <TestOverflowItem id={'3'} priority={3} groupId={'4'} />
-        <TestOverflowGroupDivider groupId={'4'} />
-        <TestOverflowItem id={'8'} priority={8} groupId={'5'} />
+        <OverflowItem id={'6'} priority={6} groupId={'1'}>
+          <Button>Priority 6</Button>
+        </OverflowItem>
+        <OverflowGroupDivider groupId={'1'} />
+        <OverflowItem id={'7'} priority={7} groupId={'2'}>
+          <Button>Priority 7</Button>
+        </OverflowItem>
+        <OverflowGroupDivider groupId={'2'} />
+        <OverflowItem id={'4'} priority={4} groupId={'3'}>
+          <Button>Priority 4</Button>
+        </OverflowItem>
+        <OverflowItem id={'5'} priority={5} groupId={'3'}>
+          <Button>Priority 5</Button>
+        </OverflowItem>
+        <OverflowGroupDivider groupId={'3'} />
+        <OverflowItem id={'1'} priority={1} groupId={'4'}>
+          <Button>Priority 1</Button>
+        </OverflowItem>
+        <OverflowItem id={'2'} priority={2} groupId={'4'}>
```

Comment:
Like with the Dividers story both Items 1 and 2 are overflowed at the same time when I'd only expect Item 1 to be hidden.


https://user-images.githubusercontent.com/93940821/210870688-668a66bb-6dda-49b8-90ed-b54454bfa6e4.mp4

Reply:
resolving this comment to keep focus on the original one, since they discuss the same thing